recipes/libbsc/meta.yaml

```
@@ -0,0 +1,43 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+{% set name = "libbsc" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IlyaGrebnov/libbsc/archive/{{ version }}.tar.gz
```

You can make the following change after changing the version as above.
```suggestion
  url: https://github.com/IlyaGrebnov/libbsc/archive/3dea3471251fbac72c895f42f22a1aa75b81da33.tar.gz
```